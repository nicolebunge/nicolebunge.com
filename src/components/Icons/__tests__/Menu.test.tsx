import { render } from '@testing-library/react';
import React from 'react';
import { Menu1 } from '..';

describe('Xing icon component', () => {
  it('renders correctly', () => {
    const { container } = render(<Menu1 width={24} height={24} fill="#000" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
