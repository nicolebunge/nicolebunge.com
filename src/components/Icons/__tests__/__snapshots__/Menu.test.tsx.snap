// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Xing icon component renders correctly 1`] = `
<svg
  fill="#000"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M24 6H0V2h24v4zm0 4H0v4h24v-4zm0 8H0v4h24v-4z"
  />
</svg>
`;
